.github/workflows/pubsys-exercise.yaml

```diff
@@ -0,0 +1,27 @@
+name: pubsys
+on:
+  pull_request:
+    branches: [develop]
+    paths:
+      - 'tools/**'
+jobs:
+  check:
+    runs-on: [self-hosted, linux, x64]
+    continue-on-error: true
```

Comment:
I'm not sure I understand this - we want the workflow to pass even if our checks fail?  We use continue-on-error in the build workflow only for specific matrix entries so we can get data on them but not block on them, but here, it seems like we should require passing?

Reply:
The workflow won't pass if any of the steps fail. I just don't want the workflow to stop if one of the steps fail. I'd still like to run through all the steps.

Reply:
Oh, I think I misunderstood this field. It does allow the workflow to pass if the job fails. Hmm, not sure how I can express "fail but continue".

Reply:
Talked about this offline a bit, while there are workarounds, they are the most elegant. We're fine with a workflow short-circuit if any of the steps fail so I'm just gonna get rid of `continue-on-error`